types/selenium-webdriver/bidi/evaluateResult.d.ts

```diff
@@ -0,0 +1,39 @@
+interface IEvaluateResultType {
+    SUCCESS: "success";
+    EXCEPTION: "exception";
+}
+
+const EvaluateResultType: IEvaluateResultType;
+
+declare class EvaluateResultSuccess<T> {
+    constructor(realmId: string, value: T);
+    resultType: "success";
+    realmId: string;
+    result: T;
+}
+
+declare type ExceptionShape = {
```

Comment:
Just curious, why is ExceptionShape used here? Due to reserved words in the language? Maybe use EvaluateResultException? To match what is present in JS.